### DISCUSSION TITLE
Some question about definition of foreground class

### DISCUSSION THREAD

**Discussion by cvlearn913** [MIC-DKFZ/nnUNet#975] [Q&A] [answered]
The paper says that there is at least one positive sample in each batch, so what is the definition of a positive sample or foreground? What is the distinction between foreground and background?

**Comment by FabianIsensee** [comment, marked as answer]
background is label 0, foreground is everything else :-)

**Reply by cvlearn913** [reply]
Thanks for your kindly reply!What confuses me is that when  clipping the patch is let a postive or negtive voxel as center and crop the patch around this voxel?

**Reply by FabianIsensee** [reply]
what do you mean by negative voxel?

The algorithm (roughly) is as follows:
1) pick a random train case
2) pick a random foreground class (any class > 0)
3) pick a random pixel of that class
4) crop around that pixel

**Comment by cvlearn913** [comment]
I mean negative voxel is background voxel .Anyway，thanks for your reply，I know the way where the patch comes from



发自 网易邮箱大师




---- 回复的原邮件 ----
| 发件人 | Fabian ***@***.***> |
| 日期 | 2022年03月14日 23:21 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.******@***.***> |
| 主题 | Re: [MIC-DKFZ/nnUNet] Some question about definition of foreground class (Discussion #975) |

what do you mean by negative voxel?

The algorithm (roughly) is as follows:

pick a random train case
pick a random foreground class (any class > 0)
pick a random pixel of that class
crop around that pixel

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
